packages/ui/src/domains/LandingPageBuilder/LandingPageBuilder.tsx

```
@@ -0,0 +1,2555 @@
+	PopoverTrigger,
+} from "../../components/ui/popover";
+import {
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableHeader,
+	TableRow,
+} from "../../components/ui/table";
+import Image from "next/image";
+import { throws } from "assert";
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)